readme.md

```diff
@@ -19,6 +19,20 @@ npm install flightradar24-client
 
 ### `radar(north, west, south, east)`
 
+Here they represent a geographical bounding box (in decimal degrees) with: 
+
+```
+ Northern edge latitude
```

Comment:
👍 very helpful. would you mind creating a list instead of a code block, with this format?

- `variable`: meaning...